### PR TITLE
[LMS-342][Integration] Sign out

### PR DESCRIPTION
**Pull request by mikorosales**
## Issue Link
[LMS-342](https://framgiaph.backlog.com/view/LMS-342)

## Defintion of Done
- [x] it should auto redirect to landing page once signing-out is successful
- [x] it should not be able to access home page after signing-out
- [x] it should clear cache or localstorage once signing-out is successful

## Steps to reproduce
make necessary migrations first, refer to this [PR](https://github.com/framgia/sph-lms/pull/53)
1. You can try logging in from here http://localhost:3000/ by clicking log in
2. Sign in using seeded credentials ( admin@gmail.com || password )
3. Logout

## Affected Components / Functionalities / Page
n/a

## Test Cases
Please mark while checking my PR
- [x] Once logged out users is directed to the landing page '/'
- [x] Once logged out, users cannot access '/home'
- [x] Once logged out, token is destroyed, you may check cache

## Notes
In this PR I also changed:
../hooks/useAuthSignin.ts line 28
`username: email, -> email: email,`

## Screenshots
signin-logout-demo
image

### CODE REVIEW

**Review by ragg09-sunasterisk — COMMENTED** [review]
please move logout button in the dropdown of profile
image

File: client/src/shared/hooks/useSignOut.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      await axios.post(
+        `${process.env.NEXT_PUBLIC_API_BASE_URL}auth/sign-out`,
+        null,
+        {
+          headers: {
+            Authorization: `Token ${token}`
+          }
+        }
+      );
+      // Remove token from localStorage
```
Comment (ragg09-sunasterisk) [inline client/src/shared/hooks/useSignOut.ts:21]: - remove comment
- also remove from local storage the ff:
-- user_full_name
-- user_username
-- user_email

**Review by Marielleregner7 — APPROVED** [review]
**APPROVED**
- [x]  logout button is transferred in the Dropdown component. 
- [x] able to sign out and got redirected to landing page were I could sign in again.
- [x] once logout users cannot access pages other than sign in and landing page.
- [x] No ESLint warnings or errors.

Comments: 4
Test time: 60 mins
Iterations: 1

Proof:
logoutsirmiko
image

**Review by ragg09-sunasterisk — APPROVED** [review]

**Comment by ragg09-sunasterisk** [comment]
Comments: 4
Iterations: 1
signout_integ

**Pull request merged**